osquery/distributed/distributed.cpp

```diff
@@ -117,11 +119,56 @@ Status Distributed::serializeResults(std::string& json) {
     doc.add(result.request.id, arr, queries_obj);
     doc.add(result.request.id, result.status.getCode(), statuses_obj);
     doc.add(result.request.id, result.message, messages_obj);
+
+    RecursiveLock lock(performance_mutex_);
+    auto obj = doc.getObject();
+    if (performance_.count(result.request.id) > 0) {
+      auto perf = performance_[result.request.id];
+      obj.AddMember("executions",
+                    static_cast<uint64_t>(perf.executions),
+                    obj.GetAllocator());
+      obj.AddMember("last_executed",
+                    static_cast<uint64_t>(perf.last_executed),
+                    obj.GetAllocator());
+      obj.AddMember("output_size",
+                    static_cast<uint64_t>(perf.output_size),
+                    obj.GetAllocator());
```

Comment:
Similar question: For `output_size` since the result data is returned with the request I think this could essentially be calculated from the same request that we output this in. Do you think it adds value to include it like this?

Reply:
I think this has even less reason than `last_executed` to be included, I'll remove it.